server/utils/reportCategories.js

```diff
@@ -0,0 +1,16 @@
+const reports = [
```

Comment:
Move this file and  reports.js file `seed-data` folder

Reply:
I have moved them.